### ISSUE TITLE
User is not able to specify where rbt commands are executed from

### ISSUE THREAD

**Issue by abaniuszewicz** [abaniuszewicz/rbtools-context-menu#1]
User should be able to specify exact directory where `rbt` commands are being issued. Right now this defaults to directory where _.exe_ file lies and prevents commands from being executed.

```
4/2/2021 3:20:07 PM: C:\Users\USERNAME\Desktop\Release>rbt post xyz
4/2/2021 3:20:08 PM: ERROR: The current directory does not contain a checkout from a supported source code repository.
```

**Comment by abaniuszewicz** [comment]
Control added in a79702234efc8da7e8184c3d847693cb7e31a5fb

**Comment by abaniuszewicz** [comment]
Fixed on 5a298a7
